.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
   - sudo apt-get install -y build-essential libcap-dev
 install:
   - pip install tox
+  - pip install -r test_requirements.txt
```

Comment:
I added it there as well.  I think I could probably just remove this line entirely come to think of it, since I'm also specifying it in tox.